modules/mssql/mssql.go

```diff
@@ -0,0 +1,98 @@
+package mssql
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const defaultImage = "mcr.microsoft.com/mssql/server:2022-latest"
```

Comment:
latest docker image - [source](https://hub.docker.com/_/microsoft-mssql-server)

Reply:
BTW could you use a fixed version instead of latest? We do not want users to see changes in the modules they use because the latest tag was changed by the Microsoft folks 🙏

Reply:
sure can - good call 😄